generic/stress-ng.py

```diff
@@ -198,11 +198,10 @@ def test(self):
                       "\n".join(ERROR))
 
     def tearDown(self):
-        if 'filesystem' in self.class_type:
-            process.run("umount %s" % self.loop_dev, ignore_status=True,
-                        sudo=True)
-            process.run("losetup -d %s" % self.loop_dev, ignore_status=True,
-                        sudo=True)
+        if hasattr(self, 'loop_dev') and os.path.exists(self.loop_dev):
+            process.run("umount %s" % self.loop_dev, ignore_status=True, sudo=True)
+            process.run("losetup -d %s" % self.loop_dev, ignore_status=True, sudo=True)
```

Comment:
@disgoel please take care of pep8 errors


```
# pycodestyle test.py 
test.py:80:80: E501 line too long (82 > 79 characters)
test.py:117:80: E501 line too long (81 > 79 characters)
test.py:124:80: E501 line too long (87 > 79 characters)
test.py:173:80: E501 line too long (86 > 79 characters)
test.py:184:80: E501 line too long (86 > 79 characters)
test.py:202:80: E501 line too long (83 > 79 characters)
test.py:203:80: E501 line too long (87 > 79 characters)

```

Reply:
fixed pep8 errors